diff --git a/node_modules/@noir-lang/barretenberg/dest/wasm/barretenberg_wasm.js b/node_modules/@noir-lang/barretenberg/dest/wasm/barretenberg_wasm.js
index 91a2bb7..dc794db 100644
--- a/node_modules/@noir-lang/barretenberg/dest/wasm/barretenberg_wasm.js
+++ b/node_modules/@noir-lang/barretenberg/dest/wasm/barretenberg_wasm.js
@@ -15,7 +15,7 @@ async function fetchCode() {
         return await (0, util_1.promisify)(fs_1.readFile)(__dirname + '/barretenberg.wasm');
     }
     else {
-        const res = await fetch('/barretenberg.wasm');
+        const res = await fetch(new URL('./barretenberg.wasm', import.meta.url));
         return Buffer.from(await res.arrayBuffer());
     }
 }
diff --git a/node_modules/@noir-lang/barretenberg/dest/wasm/worker_factory.js b/node_modules/@noir-lang/barretenberg/dest/wasm/worker_factory.js
index bfa87e5..d5baec9 100644
--- a/node_modules/@noir-lang/barretenberg/dest/wasm/worker_factory.js
+++ b/node_modules/@noir-lang/barretenberg/dest/wasm/worker_factory.js
@@ -5,7 +5,7 @@ const threads_1 = require("threads");
 const log_1 = require("../log");
 async function createWorker(id, module, initial, timeout = 5 * 60 * 1000) {
     const debug = (0, log_1.createDebugLogger)(`bb:wasm${id ? ':' + id : ''}`);
-    const thread = await (0, threads_1.spawn)(new threads_1.Worker('./worker.js'), { timeout });
+    const thread = await (0, threads_1.spawn)(new threads_1.Worker(new URL('./worker.js', import.meta.url)), { timeout });
     thread.logs().subscribe(debug);
     await thread.init(module, initial);
     return thread;
